web/src/stream_ui_updates.ts

```diff
@@ -239,11 +253,13 @@ export function update_default_stream_and_stream_privacy_state($container) {
     );
 
     // If the default stream option is checked, the private stream options are disabled.
-    const is_default_stream = $default_stream.find("input").prop("checked");
+    const is_default_stream = util.the($default_stream.find("input")).checked;
```

Comment:
Why the change from `.prop` here?

Reply:
`@types/jquery` gives up on `prop`; it just annotates it `(propertyName: string) => any`. I asked Anders about this and he suggested I replace `….prop("checked")` with `the(…).checked`.